src/common/SurgeSynthesizerIO.cpp

```diff
@@ -200,6 +201,20 @@ void SurgeSynthesizer::loadRaw(const void* data, int size, bool preset)
    halt_engine = false;
    patch_loaded = true;
    refresh_editor = true;
+	if (patchid < 0)	//category.size() == 0)
+	{//JS: it is a setchunk so I look up the set current category and patch
+	    int cnt = storage.patch_list.size();
+		string name = storage.getPatch().name;
```

Comment:
Then indentation goes shifts suddenly 3 spaces right for no reason.

Reply:
I am not sure how to setup VS2017 to tab correctly for you guys.